src/create-config.ts

```diff
@@ -193,7 +189,7 @@ const main = async () => {
     const chainId = chainNameToChainId[chainName];
     //TODO: Add RrpBeaconServer and DapiServer contracts based on chain
     const RrpBeaconServer = '';
-    const DapiServer = '';
+    const DapiServer = operationsRepository.documentation.chains[chainName].contracts.DapiServer;
```

Comment:
This should come out of the `/chains` shouldn't it?

Reply:
its posted in both places but your right `/chains` would be better